packages/terra-spacer/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Spacer
+
```

Comment:
Lets update this description. It should read what it is and how it works, not why we need it.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-4a98913914f72896bd7b2c1918412f03R3